<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="one">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="three">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="four">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="five">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="six">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seven">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eight">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nine">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ce">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jia">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>＋</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jian">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>－</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cheng">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chu">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equal">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>112</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>＝</string>
    </property>
   </widget>
   <widget class="QLabel" name="dis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>681</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dian">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>·</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>数字</string>
    </property>
    <addaction name="num_1"/>
    <addaction name="num_2"/>
    <addaction name="num_3"/>
    <addaction name="num_4"/>
    <addaction name="num_5"/>
    <addaction name="num_6"/>
    <addaction name="num_7"/>
    <addaction name="num_8"/>
    <addaction name="num_9"/>
    <addaction name="num_0"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>符号</string>
    </property>
    <addaction name="dian_shortcut"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="equal_shortcut"/>
    <addaction name="jia_shortcut"/>
    <addaction name="jian_shortcut"/>
    <addaction name="cheng_shortcut"/>
    <addaction name="chu_shortcut"/>
    <addaction name="back_shortcut"/>
    <addaction name="ce_shortcut"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="num_1">
   <property name="text">
    <string>1</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="num_2">
   <property name="text">
    <string>2</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="num_3">
   <property name="text">
    <string>3</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="num_4">
   <property name="text">
    <string>4</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="num_5">
   <property name="text">
    <string>5</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="num_6">
   <property name="text">
    <string>6</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
  <action name="num_7">
   <property name="text">
    <string>7</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
  </action>
  <action name="num_8">
   <property name="text">
    <string>8</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
  </action>
  <action name="num_9">
   <property name="text">
    <string>9</string>
   </property>
   <property name="shortcut">
    <string>9</string>
   </property>
  </action>
  <action name="num_0">
   <property name="text">
    <string>0</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>数字</string>
   </property>
  </action>
  <action name="dian_shortcut">
   <property name="text">
    <string>点</string>
   </property>
   <property name="shortcut">
    <string>.</string>
   </property>
  </action>
  <action name="equal_shortcut">
   <property name="text">
    <string>等于</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="jia_shortcut">
   <property name="text">
    <string>加</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="jian_shortcut">
   <property name="text">
    <string>减</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="cheng_shortcut">
   <property name="text">
    <string>乘</string>
   </property>
   <property name="shortcut">
    <string>*</string>
   </property>
  </action>
  <action name="chu_shortcut">
   <property name="text">
    <string>除</string>
   </property>
   <property name="shortcut">
    <string>/</string>
   </property>
  </action>
  <action name="back_shortcut">
   <property name="text">
    <string>退格</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="ce_shortcut">
   <property name="text">
    <string>清除</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
